### ISSUE TITLE
Events not triggering in a Cordova application

### ISSUE THREAD

**Issue by tehmaestro** [simplabs/ember-simple-auth#652]
Hi, using a Cordova application built with Ember JS and it seems that the events don't get triggered.
 I think this line `this.trigger('sessionAuthenticationSucceeded');` does nothing. It never reaches the application route's action `sessionAuthenticationSucceeded` . It is the same for sessionInvalidationSucceeded ...
```
   /routes/applications.js

   actions: {
        sessionAuthenticationSucceeded: function() {
           console.log("something"); //never reached
        }
    }
```

  Anyone else having these problems?

**Comment by marcoow** [comment]
Which version are you using?

**Comment by tehmaestro** [comment]
```
   DEBUG: Ember                   : 1.11.3
   DEBUG: Ember Data              : 1.0.0-beta.15
   DEBUG: jQuery                  : 1.11.2
   DEBUG: Ember Simple Auth       : 0.7.2
   DEBUG: Ember Simple Auth Torii : 0.7.3
```

**Comment by marcoow** [comment]
Can you update to the latest versions?

**Comment by tehmaestro** [comment]
Not really, I realize I should be on the latest version, but it is really weird that it doesn't work only when using Cordova. I am testing multiple combinations, trying to see what's causing this right now.

**Comment by tehmaestro** [comment]
I can confirm that adding the `document.addEventListener('deviceready', function () {` to listen for the deviceready event of Cordova, makes the event triggering fail.

  I even included the cordova js and plugins js files, but as soon as I add the listener, it breaks.

**Comment by tehmaestro** [comment]
After trying several combinations, I discovered that the problem is with placing the deviceready event inside the application route's beforeModel. I moved that to an initializer, which `deferReadiness` until the cordova plugins are ready ... this seems to work now.
  I know this is not ideal, but it works, I don't know why exactly.

 Anyway, it is not related to simple auth, that's just how I discovered this. I'm closing this issue. Thanks.
